package.json

```diff
@@ -23,6 +23,11 @@
     "underscore": "^1.8.3"
   },
   "devDependencies": {
-    "nodemon": "^1.9.1"
+    "bourbon": "^4.2.6",
+    "bourbon-neat": "^1.7.4",
```

Comment:
Hey @billiamliu, any reason why you're creating a list of devDependencies instead of just lumping them all into the dependencies?